ait/core/server/handler.py

```diff
@@ -72,8 +71,8 @@ def __init__(self, input_type=None, output_type=None, **kwargs):
         tlm_dict = tlm.getDefaultDict()
         if self.packet not in tlm_dict:
             msg = 'PacketHandler: Packet name {} not present in telemetry dictionary'.format(self.packet)
-            msg += ' Available packet types are {}'.format(tlm_dict.keys())
-            raise ValueError(msg)
```

Comment:
I think we should still be raising `ValueError` here. If the configured packet type isn't present in the config the handler can't do anything.